src/Network/Network/Generated/LoadBalancer/OutboundRule/AddAzureRmLoadBalancerOutboundRuleConfigCommand.cs

```diff
@@ -35,16 +39,51 @@
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "LoadBalancerOutboundRuleConfig", DefaultParameterSetName = "SetByResource", SupportsShouldProcess = true), OutputType(typeof(PSLoadBalancer))]
+    [Cmdlet(VerbsCommon.Add, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "LoadBalancerOutboundRuleConfig", DefaultParameterSetName = "SetByResourceParent", SupportsShouldProcess = true), OutputType(typeof(PSLoadBalancer))]
```

Comment:
What are the changes to this cmdlet for?

Reply:
[as above](https://github.com/Azure/azure-powershell/pull/9130#discussion_r286885464) for all LB related stuff